README.md

```diff
@@ -275,6 +275,8 @@ For those cases, additional properties (apart from `message`) are included:
     * `proxy` [`<String>`][] - The full URL of an http or https proxy to pass through
     * `ignoreRetryableErrors` [`<Boolean>`][] - Do not emit "errors" that are retry-able. Typically, theses are
       temporary connection-based errors. **Default:** `true`
+    * `sendUserAgent` [`<Boolean>`][] - If `true`, the `user-agent` request header is sent. CORS issues when using
+      the logger in a browser may require setting this to `false`. **Default:** `true`
```

Comment:
two things here:

1. I'd suggest clarifying that this option controls the sending of the _client_ user-agent (our library `user-agent` string). When this setting is `false` in the browser context, the _browser_ user-agent is still getting sent implicitly by the browser, correct?
2. This doesn't have anything to do with CORS afaik, the issue of "Refused to send..." is a separate thing entirely.

Reply:
Yeah, that needs clarity.   You're right about "CORS" technically, but people are thinking that it's CORS related.  I'll clean it up.